.github/workflows/release.yml

```diff
@@ -104,11 +104,37 @@ jobs:
     uses: ./.github/workflows/sam_release_code.yml
     with:
       ARTIFACT_BUCKET_PREFIX: ${{needs.tag_release.outputs.spec_version}}
-      STACK_NAME: dev-ci
+      STACK_NAME: pfp
       TARGET_ENVIRONMENT: dev
       ENABLE_MUTUAL_TLS: true
       DEPLOY_SANDBOX: false
```

Comment:
I dont think we need deploy_sandbox as a variable anymore (its mentioned lots of places)